tests/common/reboot.py

```diff
@@ -221,9 +221,9 @@ def reboot(duthost, localhost, reboot_type='cold', delay=10,
             timeout = reboot_ctrl['timeout']
         if wait == 0:
             wait = reboot_ctrl['wait']
-        if plt_reboot_ctrl:
-            wait = plt_reboot_ctrl['wait']
-            timeout = plt_reboot_ctrl['timeout']
+            if plt_reboot_ctrl:
```

Comment:
why change indention here?

Reply:
this is not a feasible fix, it'll only overwrite timeout if `wait == 0`, however, the intention for having plt_reboot_ctrl is to give privilege for some TBs(chassis) to predefine some timeout that has more tolerance.

if we want to pass the `wait time = 5` successfully, we could remove the wait time in inventory for your sku, and 
new_timeout= plt_reboot_ctrl.get('timeout', timeout)
new_wait = plt_reboot_ctrl.get('wait', wait)

or simply change the `wait` in inventory to 5 or a more reasonable number that works generally.

Reply:
**the wait has kept for small duration intentionally to verifythe test -for reload configuration checks. The test is simulated to perform config reload before the dut is stablize  and expect a check to the proper message.
 ***The plt_reboot_ctrl should only apply, if the (wait) time is not explicitly provided for a known objective